.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
can you add this to .gitignore? @harshadmanglani

Reply:
Done. @michaelcspeed